src/views/tutorial-view/utils/get-collection-context.ts

```diff
@@ -1,27 +1,24 @@
 import {
-  ProductOption,
+  Collection as ClientCollection,
   TutorialFullCollectionCtx as ClientTutorialFullCollectionCtx,
 } from 'lib/learn-client/types'
 import { CollectionContext } from '..'
 import { formatCollectionCard } from '../components/featured-in-collections/helpers'
 
 export function getCollectionContext(
-  productSlug: ProductOption,
-  filename: string,
+  currentCollection: ClientCollection,
```

Comment:
Refactoring this a bit since we get the 'current collection' with the changes above, we don't need to 'sniff it out' via featured collections now.